doc-surrealql_versioned_docs/version-latest/statements/create.mdx

```diff
@@ -361,6 +363,41 @@ The `PARALLEL` keyword can be used to specify that the statement should be execu
 CREATE person:26, CREATE person:27 PARALLEL; 
 ```
 
+
+### `VERSION`
+
+<Since v="v2.0.0" />
+
+If you are using [SurrealKV as the storage engine](docs/surrealdb/cli/start#surrealkv-beta), when creating a record you can specify a version for each record. This is useful for time-travel queries. You can query a specific version of a record by using the `VERSION` clause. 
+
+The `VERSION` clause is always followed by a [datetime](/docs/surrealql/datamodel/datetimes) and when the specified timestamp does not exist, an empty array is returned.
+
+:::note
+__Note:__ The `VERSION` clause is currently in alpha and is subject to change. We do not recommend this for production.
+:::
+
+```surql
+-- Create a record for user:john at 8:00AM
```

Comment:
The other thing I'd recommend for the examples (both CREATE and SELECT) is that you can see the current state of all the records at a certain time.

e.g. CREATE user VERSION some_time_stamp;
SELECT * FROM user (shows one record)
CREATE user VERSION some_time_stamp;
SELECT * FROM user VERSION previous_time_stamp (still shows one record even though SELECT * FROM user without VERSION would show two)